gnmi_cli_py/py_gnmicli.py

```diff
@@ -403,19 +405,23 @@ def gen_request(paths, opt, prefix):
     yield mysubreq
 
 
-def subscribe_start(stub, options, req_iterator):
+def subscribe_start(stub, options, req_iterator, exit_early):
   """ RPC Start for Subscribe reqeust
   Args:
       stub: (class) gNMI Stub used to build the secure channel.
       options: (dict) Command line argument passed for subscribe reqeust.
       req_iterator: gNMI Subscribe Request from gen_request.
+      exit_early: create TCP connections with gNMI server without explicitly 
+                  closeing it to simulate memory spike on server side.
   Returns:
       Start Subscribe and printing response of gNMI Subscribe Response.
   """
   metadata = [('username', options['username']), ('password', options['password'])]
   max_update_count = options["update_count"]
   try:
       responses = stub.Subscribe(req_iterator, options['timeout'], metadata=metadata)
+      if exit_early:
```

Comment:
>exit_early [](http://example.com/codeflow?start=9&length=10)

The behavior you want to achieve: when there is missing path in the request, response will get some error, then you exit early. If there is no missing path in the request, no need to exit.

Reply:
As comments posted on the below, if user specifies the option `--trigger_mem_spike`, then there is a while loop which continuoulsy creates a new channel, build a stub and then call the streaming RPC `subscribe(...)` method implemented in the gNMI server side.

Since this option `--trigger_mem_spike` just was used to simulate the high memory usage on the gNMI server side, we do not need get the actual response from gNMI server side. If user specifies `--trigger_mem_spike`, then `exit_early = True` and `susbscribe(...)` method will exit directly instead of waiting for the response from gNMI server. That's why the flag `exit_early` was used at here.